### PR TITLE
feat(vulnerability): add primary URLs

### PR DESCRIPTION
**Pull request by knqyf263**
In place of https://github.com/aquasecurity/trivy/pull/709

### CODE REVIEW

File: pkg/vulnerability/vulnerability.go [hunk 1]
```diff
@@ -25,6 +25,19 @@ const (
...
+		vulnerability.Debian:           {"http://www.debian.org", "https://www.debian.org"},
+		vulnerability.Ubuntu:           {"http://www.ubuntu.com", "https://usn.ubuntu.com"},
+		vulnerability.RedHat:           {"https://access.redhat.com"},
+		vulnerability.OpenSuseCVRF:     {"http://lists.opensuse.org", "https://lists.opensuse.org"},
+		vulnerability.SuseCVRF:         {"http://lists.opensuse.org", "https://lists.opensuse.org"},
+		vulnerability.OracleOVAL:       {"http://linux.oracle.com/errata", "https://linux.oracle.com/errata"},
+		vulnerability.NodejsSecurityWg: {"https://www.npmjs.com", "https://hackerone.com"},
+		vulnerability.RubySec:          {"https://groups.google.com"},
+	}
+)
```
Comment (simar7) [inline pkg/vulnerability/vulnerability.go:39]: Now I understand what you had in mind. This is much better and quite impressive to use for fallback. Nice!

File: pkg/rpc/convert.go [hunk 2]
```diff
@@ -78,27 +78,6 @@ func ConvertToRPCLibraries(libs []deptypes.Library) []*common.Library {
...
-				Title:       vuln.Title,
-				Description: vuln.Description,
-				Severity:    severity.String(),
-				References:  vuln.References,
-			},
-		})
-	}
-	return vulns
-}
-
```
Comment (simar7) [inline pkg/rpc/convert.go:101]: Why is this part of the diff? I thought we removed it already right?

**Review by simar7 — APPROVED** [review]
mucho mejor! 👍

File: pkg/rpc/convert.go (hunk 2 above)
Comment (knqyf263) [reply pkg/rpc/convert.go:101]: Yes, I think so, too. But I found it was not used and just removed it.

File: pkg/vulnerability/vulnerability.go (hunk 1 above)
Comment (knqyf263) [reply pkg/vulnerability/vulnerability.go:39]: Thank you for understanding! Yes, I thought it was pretty easy to support other vulnerability IDs as well as AVD and didn't find any reason not to do that. Sometimes it's hard to explain an idea with words and I'm sorry I couldn't tell it well😁 The source code says more than words.

**Pull request merged**